cmd/influxd/launcher/cmd.go

```diff
@@ -66,26 +69,45 @@ func setCmdDescriptions(cmd *cobra.Command) {
 `
 }
 
-func cmdRunE(ctx context.Context, o *InfluxdOpts) func() error {
+func cmdRunE(o *InfluxdOpts) func() error {
```

Comment:
this was better with a passed in context.   prior to this change, the caller could cancel the entire call chain.